internal/test_cases/geopos_test_case.go

```diff
@@ -0,0 +1,69 @@
+package test_cases
+
+import (
+	"math"
+
+	"github.com/codecrafters-io/redis-tester/internal/data_structures/location"
+	"github.com/codecrafters-io/redis-tester/internal/instrumented_resp_connection"
+	"github.com/codecrafters-io/redis-tester/internal/resp_assertions"
+	"github.com/codecrafters-io/tester-utils/logger"
+	"github.com/codecrafters-io/tester-utils/random"
+)
+
+type GeoPosTestCase struct {
+	Key                  string
+	Locations            []location.Location
+	MissingLocationNames []string
+	// If ShouldVerifyCoordinates is true, only floating point parsing is checked for existing locations
+	ShouldVerifyCoordinates bool
```

Comment:
```suggestion
	ShouldSkipCoordinateVerification bool
```

or `ShouldSkipVerifyingCoordinates`, `ShouldOnlyVerifyCoordinatesAreFloats` etc.

(The obvious case for `GeoPosTestCase` is to verify coordinates - the skipping one seems like a edge-case, would not make it the default)